src/main/resources/jenkins/plugins/http_request/HttpRequestGlobalConfig/config.jelly

```diff
@@ -1,20 +1,18 @@
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
     <f:section title="HTTP Request">
         <f:entry title="Basic/Digest Authentication">
             <f:repeatable field="basicDigestAuthentications">
                 <table width="100%">
-                    <f:entry title="Key Name" field="keyName"
-                        help="/plugin/http_request/WEB-INF/classes/jenkins/plugins/http_request/HttpRequest/help-keyName.html">
+                    <f:entry title="Key Name" field="keyName" help="${descriptor.httpRequestHelpPath}/keyName">
                         <f:textbox name="basicDigestAuthentication.keyName"/>
-                    </f:entry>  
-                    <f:entry title="Username" field="userName"
-                        help="/plugin/http_request/WEB-INF/classes/jenkins/plugins/http_request/HttpRequest/help-userName.html">
+                    </f:entry>
+                    <f:entry title="Username" field="userName" help="${descriptor.httpRequestHelpPath}/userName">
                         <f:textbox name="basicDigestAuthentication.userName" />
-                    </f:entry>  
-                    <f:entry title="Password" field="password"
-                        help="/plugin/http_request/WEB-INF/classes/jenkins/plugins/http_request/HttpRequest/help-password.html">
+                    </f:entry>
+                    <f:entry title="Password" field="password" help="${descriptor.httpRequestHelpPath}/password">
```

Comment:
These fields are nested inside the `BasicDigestAuthentication` class, they are not inside the `HttpRequestGlobalConfig`.